vale/dictionary.jsonnet

```
@@ -43,6 +43,7 @@ local newWord(word, affixes, po) = {
     newWord('Codespaces', '', 'noun') { description: 'GitHub Codespaces https://docs.github.com/en/codespaces/overview', product: true },
     newWord('composable', '', 'adjective'),
     newWord('configure', 'mpDS', 'verb'),
+    newWord('contentful', '', 'adjective'),
```

Can you add a description field to this one?  This term has a very specific context/use. A quick Google on just `contentful` directs you to the CMS and not to LCP.  Will it be referred to as LCP? Do we need to define an abbreviation as well?

@jdbaldry for something like this, do we have any mechanism in place to ensure that a word like `contentful` stays within the whole phrase it blongs to... like `Largest Contentful Paint`... and isn't used alone?

---

Nothing yet but I could implement something that would achieve that I think.

---

I don't know how necessary it would be.  This was a "hmmm I wonder" question. Maybe something to think about if it becomes an issue in the future :-)

---

@clayton-cornell would something like this work? 🤓 

```suggestion
    newWord('contentful', '', 'adjective') { description: 'Having content. Used in Web Vital metrics, such as Largest Contentful Paint: https://web.dev/articles/lcp' },
```

---

Yeah that puts it in context.